docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-advanced-operation/index.md

```diff
@@ -41,6 +42,42 @@ dbt run --select +snowplow_mobile.screen_views --vars "{'models_to_run': '$(dbt
 </TabItem>
 </Tabs>
 
+## Custom re-running of specific models
+
+There may be times where you need to re-run some or all of the models that for a period of data, potentially due to incorrect supplemental data brought into a model, or an error that was made in your model code itself. There are two methods to do this; one is simpler but must complete in a single run, the second is more complex but be processed over multiple runs. Either will work for any custom models assuming they were built using the [snowplow incremental materialization](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-incremental-materialization/index.md).
+
+:::danger
+
+Both methods are only suitable if the value of your source data in your `upsert` and date keys have not changed (e.g. `page_view_id` and `derived_tstamp` for the `snowplow_web_page_views` model). If this is not the case, or your custom models are not built using this approach, there is no choice but to run a full refresh of the model.
+
+:::
+### Option 1: Altering the look back window
+
+As defined in the [standard run incremental logic](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-incremental-logic/#state-4-standard-run) the lower limit for the events to be processed in the models is `max_last_success - snowplow__lookback_window_hours`. If you increase the value of `snowplow__lookback_window_hours` to a number that goes beyond the period you wish to re-run from, then all events from that time will be reprocessed and fed through all models in the package.
+
+For example, if your last run success was `2022-10-30 13:00:00` and you needed to reprocess events from `2022-10-25 02:00:00` you would set your `snowplow__lookback_window_hours` to `137` (5 days × 24 hours + 11 hours, and 6 hours of an additional buffer the look back window would usually provide). This will reprocess all the events in a single run, which may be larger than the value you have set in `snowplow__backfill_limit_days`. If you wish to avoid going over the backfill limit you have set please use option 2. Don't forget to change your value back once the run has completed!
```

Comment:
```suggestion
For example, if your last run success was `2022-10-30 13:00:00` and you needed to reprocess events from `2022-10-25 02:00:00`, you would set your `snowplow__lookback_window_hours` to `137` (5 days × 24 hours + 11 hours, and 6 hours of an additional buffer the look back window would usually provide). This will reprocess all the events in a single run, which may be larger than the value you have set in `snowplow__backfill_limit_days`. If you wish to avoid going over the backfill limit you have set, please use option 2. Don't forget to change your value back once the run has completed!
```